plugins/community/timostamm-protobuf-ts/v2.8.2/buf.plugin.yaml

```diff
@@ -0,0 +1,17 @@
+version: v1
+name: buf.build/community/timostamm-protobuf-ts
```

Comment:
The import is a tad long: `@buf/acme_paymentapis.community_timostamm-protobuf-ts`

Not suggesting we change it, but hopefully most users will copy/paste this.